templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,12 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 54**
+
+* [`POST /register`](/api/register-queue): Added `realm_giphy_rating`
+  and `giphy_rating_options` fields.
+* `PATCH /realm`: Added `giphy_rating` field.
```

Comment:
Just a bit of terminology -- for `PATCH /realm`, it's a "parameter" that we added, not a field in a response.